velox/functions/prestosql/coverage/data/all_scalar_functions.txt

```diff
@@ -207,8 +233,8 @@ regexp_extract_all
 regexp_like
 regexp_replace
 regexp_split
-regress
```

Comment:
why is this removed?

```
> show functions like 'regress';
 Function | Return Type |        Argument Types         | Function Type | Deterministic | Description | Variable Arity | Built In | Temporary | Language
----------+-------------+-------------------------------+---------------+---------------+-------------+----------------+----------+-----------+----------
 regress  | double      | map(bigint,double), Regressor | scalar        | true          |             | false          | true     | false     |
(1 row)
```

Reply:
Apologies, I did not find it in ```SHOW FUNCTIONS``` output and this is what I see:
```
presto:system> show functions like 'regress';
 Function | Return Type | Argument Types | Function Type | Deterministic | Description | Variable Arity | Built In | Temporary | Language
----------+-------------+----------------+---------------+---------------+-------------+----------------+----------+-----------+----------
(0 rows)
```
Will recheck and update the PR.

Reply:
These are defined in presto-ml/src/main/java/com/facebook/presto/ml/MLFeaturesFunctions.java

Reply:
Thanks, yes I found them here, added back ```regress``` and ```features``` and updated the coverage map.